mkt/access/migrations/0003_add_contenttools_groups.py

```diff
@@ -1,8 +1,9 @@
 from django.db import migrations
+from django.db.models.loading import get_model
 
 
 def create_addon_groups(apps, schema_editor):
-    Group = apps.get_model('Access', 'Group')
```

Comment:
It's actually not an app at all in the Django 1.7 sense, hence tests failing and the below line being added.